packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,750 @@
+import {configure} from '../../../page-objects/configurator';
+import {
+  InterceptAliases,
+  interceptSearch,
+  mockQuerySuggestions,
+} from '../../../page-objects/search';
+import {scope} from '../../../reporters/detailed-collector';
+import {SearchBoxActions as Actions} from './search-box-actions';
+import {SearchBoxExpectations as Expect} from './search-box-expectations';
+
+const engineId = 'quantic-search-box-engine';
+const recentQueriesLSKey = `LSKey[c]${engineId}_quantic-recent-queries`;
+
+interface StandaloneSearchBoxOptions {
+  engineId: string;
+  placeholder: string;
+  withoutSubmitButton: boolean;
+  numberOfSuggestions: number;
+  textarea: boolean;
+  disableRecentQuerySuggestions: boolean;
+}
+
+const expectedAriaLiveMessage = (suggestionsCount: number) => {
+  return `${suggestionsCount} search suggestions found, to navigate use up and down arrows.`;
+};
+
+const variants = [
+  {variantName: 'default', textarea: false},
+  {variantName: 'expandable', textarea: true},
+];
+
+function setRecentQueriesInLocalStorage(recentQueries: String[]) {
+  window.localStorage.setItem(
+    recentQueriesLSKey,
+    JSON.stringify(recentQueries)
+  );
+}
```

Comment:
question here, when we use the local storage in tests, does the storage memory of the browser resets every time we run the tests?

Reply:
Nope it doesn't reset automatically.

Reply:
ok then should we not clear the storage before setting the item?

Reply:
Cypress absolutely **should** clear localStorage between tests?
Are you sure it doesn't?